### PR TITLE
Change gate job to check all roles

### PR DESCRIPTION
**Pull request by vkondula**
Gate job using ansible lint is now checking all
roles instead of playbooks from examples.

Refactor roles to match new requirements.

### CODE REVIEW

File: roles/ovirt-collect-logs/tasks/main.yml [hunk 1]
```diff
@@ -27,53 +27,14 @@
...
-
-- name: Dump logs from all failed services
-  shell: "systemctl -t service --failed --no-legend | awk '{print $1}'
-            | xargs -r -n1 journalctl -u
-            > {{ ovirt_collect_logs_tmp_dir }}/services.txt 2>&1"
+- name: Dump system information using shell commands
+  shell: "{{ item.value }} &> {{ ovirt_collect_logs_tmp_dir }}/{{ item.key }}.txt"
+  with_dict: "{{ ovirt_collect_logs_shell_commands }}"
   ignore_errors: true
+  register: results
```
Comment (lukas-bednar) [inline roles/ovirt-collect-logs/tasks/main.yml]: I think this line is redundant

**Comment by lukas-bednar** [comment]
:+1:  looks good to me ...

@pkubica @StLuke  please take a look at that ...

File: roles/ovirt-collect-logs/tasks/main.yml (hunk 1 above)
Comment (vkondula) [inline roles/ovirt-collect-logs/tasks/main.yml]: You are right. Removed

**Comment by StLuke** [comment]
:+1:

**Pull request merged**